### DISCUSSION TITLE
Name format

### DISCUSSION THREAD

**Discussion by rameshkumar70** [pocketbase/pocketbase#4416] [Q&A] [answered]
Hi, how do you name a 
1) (base) collection? 
2) joining collection (for many-to-many relation)?
Are there any particular formats one should follow?

How do you name a relation field in a collection? Are there any restrictions on the format of the name?

**Comment by ganigeorgiev** [comment, marked as answer]
Personally I prefer `camelCase` for both collection and field names.

For the collection name I usually use a plural form (eg. `users`) and for the relation fields - singular or plural depending on the type of relation (eg. `user` for single or `users` for multiple).

Another user also posted a similar question in the past with a poll in https://github.com/pocketbase/pocketbase/discussions/3254.

**Reply by rameshkumar70** [reply]
Thank you sir
